go/indexer/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# @eth-optimism/indexer-service
```

Comment:
No need for a changelog - instead define a `package.json` and our changeset system will take care of it.